waterloop-site/src/static/copy/SustainableTech/FAQ.json

```diff
@@ -0,0 +1,24 @@
+[
+    {
+        "question": "Where will the event take place?",
+        "answer": "The Green Tech event will be fully online. Sign-up through this ",
+        "answerPart2": "!",
+        "link": true,
+        "url": "https://tinyurl.com/uwgreentech"
+    },
+    {
+        "question": "What will the event entail?",
+        "answer": "The event will kickoff with a short introduction and a quick word from *INSERT KEYNOTE SPEAKER*. Then, UWaterloo’s very own green tech teams will take the stage to talk a bit about their work in the UW community. Finally, the Waterloop team will take a few minutes to present and conclude the event.",
+        "link": false
+    },
+    {
+        "question": "Who is going to be at the event?",
+        "answer": "We are excited to have our keynote speaker *INSERT KEYNOTE SPEAKER*, several sustainable tech teams at UWaterloo (Midnight Sun, Waterloo Formula Electric, Warrior Home, and Sustainable Development Solutions Network), and Waterloop speak at our event!",
```

Comment:
I like the way you formatted this answer (it's better than what's currently shown in the Google doc), but I feel like using the word "speaking" sounds better than "speak" at the end of this sentence, do you agree?